lib/rails/generators/alchemy/install/templates/elements.yml.tt

```diff
@@ -1,6 +1,6 @@
-# == In this configuration you setup Alchemy´s element layouts.
+# == In this configuration, you set up Alchemy's element layouts.
 #
-# For further informations please see http://guides.alchemy-cms.com/create_elements.html
+# For further information please see http://guides.alchemy-cms.com/edge/elements.html
```

Comment:
Preferable: http://guides.alchemy-cms.com/elements.html always links to the most recent version, while the versioned urls are for legacy or `edge` versions. Like the [Rails guides](http://guides.rubyonrails.org)